spec/requests/vendor_api/post_test_data_spec.rb

```diff
@@ -1,31 +1,101 @@
 require 'rails_helper'
 
-RSpec.describe 'Vendor API - POST /api/v1/test-data/regenerate', type: :request do
```

Comment:
Would it make sense to have a single file per endpoint?

Reply:
Yep, I'll break it up